daemon/cmd/root.go

```diff
@@ -43,6 +43,10 @@ func NewAgentCmd(hfn func() *hive.Hive) *cobra.Command {
 
 			if err := h.Run(logging.DefaultSlogLogger); err != nil {
 				logging.Fatal(daemonLogger, fmt.Sprintf("unable to run agent: %s", err))
+			} else {
+				// If h.Run() exits with no errors, it means the agent gracefully shut down.
+				// (There is a CI job that ensures this is the case)
+				daemonLogger.Info("All stop hooks executed successfully.")
```

Comment:
Could we instead add a "Stopped" log message after the lifecycle stop in https://github.com/cilium/hive/blob/7cbbf5e0d9f595bee6240c4e3431ea95344e6b0b/hive.go#L373?